docs/medical-api/api-reference/fhir/get-medical-record-summary-status.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Get Medical Record Summary Status"
+description: "Check if a Medical Record Summary for a patient already exists, and when it was generated."
+api: "GET /patient/:id/medical-record-status"
+---
+
+This endpoint checks whether the patient's Medical Record Summary already exists in the PDF and HTML formats.
+
+To create a Medical Record Summary, you need to use the [Start Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post).
+
+## Path Params
+
+<ParamField path="id" type="string" required>
+  The ID of the Patient.
+</ParamField>
+
+## Response
+
+Returns a JSON containing the booleans for the existence of the Medical Record Summary in PDF and HTML formats, as well as the dates when they were generated.
+
+<ResponseExample>
+
+```javascript Metriport SDK
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+
+const metriport = new MetriportMedicalApi("YOUR_API_KEY");
+
+const status = await metriport.getMedicalRecordSummaryStatus(
+  "018a80c4-292a-7486-a223-6dcbc636c44c"
```

Comment:
let's make the code examples explicit, if `018a80c4-292a-7486-a223-6dcbc636c44c` is the patient ID, this would be more useful at a glance:

```
const metriport = new MetriportMedicalApi("YOUR_API_KEY");
const metriportPatientId = "018a80c4-292a-7486-a223-6dcbc636c44c";

const status = await metriport.getMedicalRecordSummaryStatus(metriportPatientId);
```

otherwise, the dev will have to look at the SDK documentation to figure out what that param is supposed to be